yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -389,369 +195,253 @@ revision 2019-01-02 {
+        <xref target="guidelines_authors"/> suggests that authors do not restrict YANG module imports to exact revision dates.
+      </t>
+      <t>Instead, for conformance purposes (section 5.6 of
+        <xref target="RFC7950"/>), the recommended approach for defining the relationship between specific YANG module revisions is to specify the relationships outside of the YANG modules, e.g., via YANG library
+        <xref target="RFC8525"/>, YANG packages
+        <xref target="I-D.ietf-netmod-yang-packages"/>, a filesystem directory containing a set of consistent YANG module revisions, or a revision control system commit label.
+      </t>
+      <section title="Recommending a minimum revision for module imports" anchor="import">
+        <t>Although the previous section indicates that the actual relationship constraints between different revisions of YANG modules should be specified outside of the modules, in some scenarios YANG modules are designed to be loosely coupled, and implementors may wish to select sets of YANG module revisions that are expected to work together. For these cases it can be helpful for a module author to provide guidance on a recommended minimum revision that is expected to satisfy an YANG import.  E.g., the module author may know of a dependency on a type or grouping that has been introduced in a particular imported YANG module revision.  Although there can be no guarantee that all derived future revisions from the particular imported module will necessarily also be compatible, older revisions of the particular imported module are very unlikely to ever be compatible.</t>
+        <t>This document introduces a new YANG extension statement to provide guidance to module implementors on a recommended minimum module revision of an imported module that is anticipated to be compatible.  This statement has been designed to be machine-readable so that tools can parse the minimum revision extension statement and generate warnings if appropriate, but this extension statement does not alter YANG module conformance of valid YANG module versions in any way, and specifically it does not alter the behavior of the YANG module import statement from that specified in
+          <xref target="RFC7950"/>.
+        </t>
+        <t>The ietf-revisions module defines the "recommended-min-date" extension statement, a substatement to the YANG "import" statement, to allow for a "minimum recommended date" to be documented:
+          <list>
+            <t>The argument to the "recommended-min-date" extension statement is a revision date.</t>
+            <t>A particular revision of an imported module adheres to an import's "recommended-min-date" extension statement if the imported module's latest revision date satisfies, i.e., does not precede, the revision date argument of the "recommended-min-date" extension statement in the importing module.
```

Comment:
Suggested alternative wording:
A particular revision of an imported module adheres to an import's "recommended-min-date" extension statement if the imported module's revision date is equal to or later than the revision date argument of the "recommended-min-date" extension statement in the importing module.

Reply:
(and similar change in the YANG module itself where this description is repeated)

Reply:
Ack. I believe I've used text similar to the suggestion above elsewhere.

Reply:
I actually hadn't, but I was thinking about it :-). Thanks for the new text.